src/components/Main.js

```diff
@@ -46,7 +48,7 @@ const Progress = function () {
         <h3>ADD NEW BOOK</h3>
         <form action="submit" onSubmit={(e) => { submitBookToStore(e); }}>
           <input type="text" placeholder="Book Title" id="b-title" />
-          <input type="text" placeholder="Book Author" id="b-author" />
+          <input type="text" placeholder="Book Category" id="b-category" />
           <button type="submit">ADD BOOK</button>
         </form>
       </div>
```

Comment:
- [ ] Adding a new book, let us validate the input fields so that a user couldn't add empty data.

- [ ] After a user adds book info, the input fields have to be empty so that a user should add another information.

![setup-8](https://user-images.githubusercontent.com/55635977/145471797-a36f0273-65c3-4339-9371-25f715ebad6a.png)